src/module-3/online-store/scss/base/_base.scss

```diff
@@ -0,0 +1,30 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css");
```

Comment:
Are you sure you need externals here? This may contain stuff you don't need or will have to reset later in your custom styles